docs/resources/access_package_catalog_role_assignment.md

```diff
@@ -0,0 +1,61 @@
+---
+subcategory: "Identity Governance"
+---
+
+# Resource: azuread_access_package_catalog_role_assignment
+
+Manages a single catalog role assignment within Azure Active Directory.
+
+## API Permissions
+
+The following API permissions are required in order to use this resource.
+
+When authenticated with a service principal, this resource requires one of the following application roles: `EntitlementManagement.ReadWrite.All` or `Directory.ReadWrite.All`
+
+When authenticated with a user principal, this resource requires one of the following directory roles: `Identity Governance administrator` or `Global Administrator`
+
+## Example Usage
+
+```terraform
+data "azuread_user" "example" {
+  user_principal_name = "jdoe@hashicorp.com"
+}
+
+resource "azuread_access_package_catalog_role" "example" {
```

Comment:
you probably meant a datasource here:
```suggestion
data "azuread_access_package_catalog_role" "example" {
```

Reply:
@qdrop17 yes you're right, I've pushed up the correct example now